openshift/core.app.yaml

```diff
@@ -14,7 +14,13 @@ objects:
     selector:
       service: core
     strategy:
-      resources: {}
+      resources:
+        requests:
+          cpu: 20m
+          memory: 120Mi
+        limits:
+          cpu: 200m
```

Comment:
I don't know if it is the right number cause @aslakknutsen did the performance tests.